playbooks/support/README.md

```diff
@@ -1,4 +1,35 @@
-### Support
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Engineering Support @ Artsy](#engineering-support--artsy)
+  - [Process Overview](#process-overview)
+  - [On-Call Responsibilities](#on-call-responsibilities)
+  - [Responding to an incident](#responding-to-an-incident)
+    - [Step 1: Alert](#step-1-alert)
+    - [Step 2: Raise](#step-2-raise)
+    - [Step 3: Assess and Identify](#step-3-assess-and-identify)
+    - [Step 4: Address Incident](#step-4-address-incident)
+      - [Investigate --> Identify --> Escalate --> Fix](#investigate----identify----escalate----fix)
+      - [Communicate](#communicate)
+    - [Step 5: Resolve](#step-5-resolve)
+  - [Severity of Incidents](#severity-of-incidents)
+  - [Best practices](#best-practices)
+  - [Handing-off to a new shift](#handing-off-to-a-new-shift)
+  - [Helpful Resources and Playbooks](#helpful-resources-and-playbooks)
+    - [Point-people](#point-people)
+    - [Engineering team playbooks](#engineering-team-playbooks)
+    - [Non-engineering team playbooks](#non-engineering-team-playbooks)
+    - [Artsy domain information](#artsy-domain-information)
+    - [External links](#external-links)
+  - [Special cases](#special-cases)
+    - [Team members who leave](#team-members-who-leave)
+    - [Holidays](#holidays)
+  - [Examples](#examples)
+    - [Non-incidents](#non-incidents)
+    - [Incidents](#incidents)
+- [Additional Resources](#additional-resources)
```

Comment:
The ToC is helpful, but I'd remove one level of nesting if possible (do the title and "additional resources" need to be top-level sections?).